src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/CSIPReport.kt

```diff
@@ -30,14 +30,18 @@ data class CSIPReport(
   @GeneratedValue(generator = "CSIP_ID")
   val id: Long = 0,
 
+  @Column(name = "CSIP_SEQ")
+  val sequence: String? = null,
+
   @ManyToOne(fetch = FetchType.LAZY)
   @JoinColumn(name = "OFFENDER_BOOK_ID")
-  val offenderBooking: OffenderBooking?,
+  val offenderBooking: OffenderBooking,
+
+  @Column(name = "ROOT_OFFENDER_ID", nullable = false)
+  val rootOffenderId: Long,
```

Comment:
Wasn't sure how to map ROOT_OFFENDER_ID - we're not interested in this value as we can get it from the offenderBooking, but it obviously needs mapping and setting.